release-notes/v0.2.2.md

```diff
@@ -0,0 +1,44 @@
+manubot version 0.2.2 release notes
+===================================
+
+Manubot version 0.2.2 contains citation and web request enhancements.
+
+New features
+------------
+
+This release adds citation support for two additional types of identifiers (`isbn` and `wikidata`).
+
+ISBNs are the primary persistent identifier for many books, so many books no longer need to be cited by URL (see [GH79](https://github.com/greenelab/manubot/pull/79) and [GH14](https://github.com/greenelab/manubot/issues/14)).
+However, ISBN metadata is sometimes missing or erroneous, so users may need to still need to set manual CSL JSON, but Manubot can at least produce a reasonable starting template.
+Try for example `manubot cite isbn:9780062316097`.
+
+[Wikidata](https://www.wikidata.org) is a free and open knowledge base that contains many records of scholarly works.
+Wikidata can store metadata on records without any persistent identifiers, and thus can help Manubot users assign a stable identifier to works that otherwise would not have one (see [GH67](https://github.com/greenelab/manubot/issues/67) and [GH86](https://github.com/greenelab/manubot/pull/86)).
```

Comment:
Clarify that Wikidata does have its own persistent identifiers, the records do not:
```suggestion
Wikidata can store metadata on records that do not have their own persistent identifiers, and thus can help Manubot users assign a stable identifier to works that otherwise would not have one (see [GH67](https://github.com/greenelab/manubot/issues/67) and [GH86](https://github.com/greenelab/manubot/pull/86)).
```